_includes/scenarios/integrate/connector-events.md

```diff
@@ -2,12 +2,13 @@
+| consumption.attributeSucceeded                                                       | [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#LocalAttribute)             | ... an Attribute was succeeded manually or through a Request.                                                                                                                                                                          |
 | consumption.attributeUpdated                                                         | [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#LocalAttribute)             | ... an Attribute was updated manually or through a Request.                                                                                                                                                                            |
 | consumption.incomingRequestReceived                                                  | [LocalRequest]({% link _docs_integrate/data-model-overview.md %}#LocalRequest)                 | ... an incoming Request was received either by loading a Relationship Template or by receiving a Message                                                                                                                               |
 | consumption.incomingRequestStatusChanged                                             | [RequestStatusChangedEventData](#requeststatuschangedeventdata)                                | ... the status of an incoming Request has changed.                                                                                                                                                                                     |
 | consumption.messageProcessed                                                         | [MessageProcessedEventData](#messageprocessedeventdata)                                        | ... a Message was processed by Modules like the `RequestModule` or `DeciderModule`.                                                                                                                                                    |
 | consumption.outgoingRequestCreated                                                   | [LocalRequest]({% link _docs_integrate/data-model-overview.md %}#LocalRequest)                 | ... an outgoing Request was created.                                                                                                                                                                                                   |
+| consumption.outgoingRequestCreatedAndCompleted                                       | [LocalRequest]({% link _docs_integrate/data-model-overview.md %}#LocalRequest)                 | ... an outgoing Request was created and directly completed.                                                                                                                                                                            |
```

Comment:
An `consumption.outgoingRequestCreatedAndCompleted` event is fired when any request was transferred via a template. This event could either be triggered:
- by an incoming relationship creation change using the `onNewRelationship` property within the RelationshipTemplateContent
- by an incoming message using the `onExistingRelationship` property within the RelationshipTemplateContent, if the user already has a relationship with the template creator

Reply:
please use more information from this comment, I think you could just copy paste it. I hope it fits in the column :D